packages/cli/test/helpers/parse-table.ts

```diff
@@ -11,7 +11,7 @@ export function getDataFromIntro(output: string): {
   };
 }
 
-export function parseTable(output: string): string[] {
```

Comment:
This function parses a row more than a table. What do you think of `parseSpacedTableRow`?